doc/content/integrations/webhooks/webhook-templates/adding_webhook_templates.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Adding Webhook Templates"
+description: ""
+weight: 4
+---
+
+{{% tts %}} uses webhook templates from the [`lorawan-webhook-templates` Github repository](https://github.com/TheThingsNetwork/lorawan-webhook-templates/).
+
+Once you have created a new webhook template with a proper [format]({{< ref "/integrations/webhooks/webhook-templates/format.md" >}}), you can easily build it locally by following next steps:
```

Comment:
I'd say `test it locally` since you're not compiling the template or something like this.